### ISSUE TITLE
Error in seq_len(n) for 1 out of 2 runs

### ISSUE THREAD

**Issue by pwkooij** [danlwarren/RWTY#180]
I'm analyzing some MrBayes data (which is still running), to check on the progress. Previously, all was ok, but now I receive the following error for one of the runs:
```
> test.rwty.reduced$Leucocoprineae_species_20221122.nexus.run1.t.correlations
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
> test.rwty.reduced$Leucocoprineae_species_20221122.nexus.run2.t.correlations
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
Error in seq_len(n) : argument must be coercible to non-negative integer
In addition: Warning messages:
1: Computation failed in `stat_density2d()`:
bandwidths must be strictly positive 
2: In min(x, na.rm = na.rm) :
  no non-missing arguments to min; returning Inf
3: In max(x, na.rm = na.rm) :
  no non-missing arguments to max; returning -Inf
4: In max(f) : no non-missing arguments to max; returning -Inf
```
As you can see the run1 command worked fine (except for the binwidth value, but this doesn't seem important), but the run2 command gave an error.
any idea what might be happening here? If necessary I can provide more information, just don't really know where to start...

**Comment by danlwarren** [comment]
Is the run2 file still being written to by MrBayes?  It's possible the file isn't ending properly if the run is still continuing, and so isn't parsing correctly.

**Comment by pwkooij** [comment]
So what I normally do is, I copy all the mrbayes files from the ongoing run to a different folder. Then I open the .t files and add `end;` at the end of these so to pretend they are finished. In general, this works perfect, and normally doesn't give any errors. But in this case it did, not sure why. When I tried later on in the runs it was working again, and another time I got similar errors for run 1. It seems quite random to me.
